loader/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+find_package (Threads REQUIRED)
+find_package (OpenCLHeaders REQUIRED)
```

Comment:
Hi Morris,

As mentioned in the comments for this PR, when I try to build these changes I am getting a cmake error:

>   CMake Error at loader/CMakeLists.txt:2 (find_package):
>   By not providing "FindOpenCLHeaders.cmake" in CMAKE_MODULE_PATH this
>   project has asked CMake to find a package configuration file provided by
>   "OpenCLHeaders", but CMake did not find one.
> 
>   Could not find a package configuration file provided by "OpenCLHeaders"
>   with any of the following names:
> 
>     OpenCLHeadersConfig.cmake
>     openclheaders-config.cmake
> 
>   Add the installation prefix of "OpenCLHeaders" to CMAKE_PREFIX_PATH or set
>   "OpenCLHeaders_DIR" to a directory containing one of the above files.  If
>   "OpenCLHeaders" provides a separate development package or SDK, be sure it
>   has been installed.

We discussed this PR in the June 11th teleconference and agreed that the changes in this PR are good (thank you again!), but we do need to resolve this issue before merging.